xml/FrameworksIndex/msbuild-16-netcore.xml

```diff
@@ -1238,6 +1239,23 @@
       <Member Id="F:Microsoft.Build.Execution.TargetResultCode.Success" />
     </Type>
   </Namespace>
+  <Namespace Name="Microsoft.Build.FileSystem">
```

Comment:
Bunch of new msbuild types. Not sure if this is due to the mdoc update, or if the binaries repo was updated but changes weren't merged into dotnet-api-docs (master).

Reply:
@ghogen @v-alje Do you know why we are seeing these mew MSBuild types? This PR tests a new version of mdoc on the existing binaries in the master branch of the binaries repo for dotnet.

Reply:
The most recent MSBuild was built from the NuGet packages released with VS 16.8, not from the binaries repo for dotnet. It would be helpful to compare what's there with what's in the NuGet packages. Can you provide a pointer to that repo?

Reply:
Looks like Alma uploaded a newer version on Jan 5 which has some new work not yet published in the docs.  I see that the MSBuildFileSystemBase was made public in August of 2020, probably we didn't pick that change up in 16.8's NuGet packages, but it's now in the binaries.

Reply:
@gewarren These were updated as a response to the discussion on the 5th, in anticipation that they would need to be updated for the doc fix.
I can revert them back to the 16.7 binaries.

Reply:
It might make sense to submit a separate PR for the MSBuild changes and merge that so that you can see the mdoc changes more clearly.  @v-alje what do you think?

Reply:
@ghogen The updates from the 16.8 binaries and XML can be found in this comparison: https://github.com/dotnet/dotnet-api-docs/compare/smoke-test-msbuild.  
However I do see some of the remarks section stuff you were manually removing in previous updates.

Reply:
I see that this is going to be more work to handle.  I already ported all our docs changes into the MSBuild sources, but only for 16.9.  The 16.8 binaries do not have those backported docs changes.  So if we want to continue with the 16.8 changes, I have to do some work here to clean up the smoke-test-msbuild branch so that it only contains the 16.8 API changes and not all the comment work.

Reply:
@gewarren @ghogen The MSBuild binaries have been reverted to 16.7 in the binaries master branch.